lib/ASTGen/Sources/ASTGen/DeclAttrs.swift

```diff
@@ -107,8 +107,10 @@ extension ASTGenVisitor {
       switch attrKind {
       case .alignment:
         return self.generateAlignmentAttr(attribute: node)?.asDeclAttribute
+
       case .allowFeatureSuppression:
         return self.generateAllowFeatureSuppressionAttr(attribute: node, attrName: attrName)?.asDeclAttribute
+
```

Comment:
Please remove this diff. Or it will conflict with https://github.com/swiftlang/swift/pull/78019/files